tests/features/withads_search.feature

```diff
@@ -5,10 +5,16 @@ Feature: Search tests that return ads to verify telemetry
     Scenario: Telemetry reports correctly for URL search events
         Given Firefox is launched enrolled in an Experiment
         And The user searches for something that is likely to return ads
-        Then The browser reports correct telemetry for the search event
+        Then The browser reports correct telemetry for the urlbar search event
 
     @smoke
     Scenario: Telemetry reports correctly for search bar search events
         Given Firefox is launched enrolled in an Experiment
         And The user searches for something in the search bar that will return ads
-        Then The browser reports correct telemetry for the searchbar event
+        Then The browser reports correct telemetry for the searchbar search event
+
+    @smoke
+    Scenario: Telemetry reports correctly for context menu search events
+        Given Firefox is launched enrolled in an Experiment
+        And The user highlights some text and wants to search for it via the context menu
+        Then The browser reports correct telemetry for the context menu contextmenu search event
```

Comment:
q: does this need "context menu" and "contextmenu"?
Not sure if that affects the parser stuff below, re:
https://github.com/mozilla/klaatu/blob/270fc66f28c110356bdb3fce19e2987f64fb2265/tests/scenarios/test_withads_search.py#L84-L86